chainer/training/extensions/exponential_decay.py

```diff
@@ -0,0 +1,47 @@
+from chainer.training import extension
+
+
+class ExponentialDecay(extension.Extension):
+    """Trainer extension to exponentially decay an optimizer attribute.
+
```

Comment:
I am slightly wondering if the name is appropriate because if decay_rate is more than 1, this extension actually increase the value of target attribute :)